LEAF_Request_Portal/admin/templates/menu.tpl

```diff
@@ -90,6 +90,11 @@
 <script>
 $(document).ready(function() {
 
+// check if top orange banner is empty or not
+var topOrange = $('.top-orange-bar-text').text();
+if (topOrange == '') {
```

Comment:
We don't need to use jQuery to check if the portal is secure, please use {if $leafSecure == 0} flag

Reply:
Removed jquery for show/hide and expanded leafSecure flag to wrap entire orange bar.